larq/models.py

```diff
@@ -12,16 +12,17 @@ def _get_delimiter(type_="thin"):
     return "=" if type_ == "thick" else "-"
 
 
-def _count_params(weights):
+def _count_params(weights, ignore=[]):
     """Count the total number of scalars composing the weights.
 
     # Arguments
     weights: An iterable containing the weights on which to compute params
+    ignore: A list of weights to ignore
 
     # Returns
     The total number of scalars composing the weights
     """
-    return int(sum(np.prod(w.shape.as_list()) for w in weights))
+    return int(sum(np.prod(w.shape.as_list()) for w in weights if w not in ignore))
 
```

Comment:
I am confused here, where is the ignore list generated?

Reply:
It is passed in as an argument [here](https://github.com/plumerai/larq/pull/125/files/82d65496ffd9cb75cbe0ffacede5e15dc1cd07a0#diff-4e95f15be669c520c1d0e3d59bac0d33R15)